src/writeData/components/ClientCodeQueryHelper.tsx

```diff
@@ -53,8 +53,7 @@ const ClientCodeQueryHelper: FC<Props> = ({clientQuery, contentID}) => {
 
     const ast = parse(clientQuery)
     const queryBucket = getBucketsFromAST(ast)[0]
-
-    changeBucket({name: queryBucket} as Bucket)
+    queryBucket && changeBucket({name: queryBucket} as Bucket)
```

Comment:
nit: this reads a bit clearer IMO
```suggestion
    if (queryBucket) {
      changeBucket({name: queryBucket} as Bucket)
    }
```